src/NuGet.Core/NuGet.Configuration/PackageSource/PackageSourceProvider.cs

```diff
@@ -158,8 +158,8 @@ private static void AddDefaultPackageSources(List<PackageSource> loadedPackageSo
 
             foreach (var packageSource in defaultPackageSources)
             {
-                var sourceMatching = loadedPackageSources.Any(p => p.Source.Equals(packageSource.Source, StringComparison.CurrentCultureIgnoreCase));
```

Comment:
Related to https://github.com/NuGet/NuGet.Client/pull/4264/files#r708797119
Looks not really relevant anymore.